.github/actions/check_if_apple_artifacts_are_there/action.yml

```
@@ -0,0 +1,56 @@
+name: check_if_apple_artifacts_are_there
```

nit: Perhaps shorten this action / file name.

```suggestion
name: check_apple_artifacts_present
```